receiver/podmanreceiver/libpod_client_test.go

```diff
@@ -167,11 +134,75 @@ func TestStatsError(t *testing.T) {
 		Timeout: 5 * time.Second,
 	}
 
-	cli, err := newPodmanClient(zap.NewNop(), config)
+	cli, err := newLibpodClient(zap.NewNop(), config)
 	assert.NotNil(t, cli)
 	assert.Nil(t, err)
 
-	stats, err := cli.stats(context.Background())
+	stats, err := cli.stats(context.Background(), nil)
 	assert.Nil(t, stats)
 	assert.EqualError(t, err, errNoStatsFound.Error())
 }
+
+func TestList(t *testing.T) {
```

Comment:
Can the new events function also be tested?

Reply:
Tests added in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/10421/commits/429a57b5270b0c3f2ac371230c01816ec71dbb00